contrib/cirrus/integration_test.sh

```diff
@@ -63,6 +63,6 @@ else
             ;;
         *) bad_os_id_ver ;;
     esac
-    make localintegration
+    make localintegration localsystem
```

Comment:
Can we get this in a separate VM? Would require more intervention from Chris, but our full integration tests are already taking 40-ish minutes.

Reply:
Can we defer optimization? This test takes 2m40s (5 if rootless is tested serially), and we're wasting a lot more time than that with bugs that current CI is not catching.

Reply:
FWIW- I wouldn't bother with a separate VM, those take a full minute just to come up.  But running `make localsystem` would be good to do as a separate "systemtest_script" (in `cirrus.yml`), for no other reason than to separate the output.

Reply:
Is this a duplicate or stale comment? The one-step 'make localintegration localsystem' went away weeks ago; I'm confused by this.

Reply:
You're right, under 'files' tab it's correct :confused:   What tha heck github...OIC, it does say 'outdated' at the top.   I think if I hit the 'resolved' button, this thread will go away, lessee...